example/index.html

```diff
@@ -414,17 +414,32 @@ <h3 class="modal__title" id="modal-1-title">
 
   // Initial config for setting up modals
   MicroModal.init({
-    onClose: modal => console.info(`${modal.id} is hidden`),
-    onShow: modal => console.info(`${modal.id} is shown`)
+    onClose: modal => {
```

Comment:
This is meant to accept callbacks provided by the user. It is a very wrong place for animation bindings. It should all be handled internally in the `openModal` and `closeModal` methods.

Reply:
I will take a look at these methods. I missed it. We should also update our document around it on the site.